app/config/Config.scala

```diff
@@ -82,8 +82,8 @@ class Config(playConfig: Configuration) extends AwsInstanceTags with Logging {
 
   lazy val sessionId: String = UUID.randomUUID().toString
 
-  lazy val adminWhitelist: List[String] = Try {
-    playConfig.underlying.getStringList("application.admin.whitelist").asScala.toList
+  lazy val adminUsers: List[String] = Try {
+    playConfig.underlying.getStringList("application.admin.users").asScala.toList
```

Comment:
Is this going to cause problems with any existing conf files? It may be worth updating the README `Admin Permissions` section.

Reply:
I'm planning on updating the config in CODE and PROD before shipping. https://github.com/guardian/workflow-frontend/pull/245 should go out soon after which removes this config entirely too.

Reply:
If the whitelist is going to be removed, is it worth putting in the extra effort to rename the whitelist name in the conf files?

Reply:
I think so. #245 will go out about 30 mins afterwards to be sure Workflow is working as it should in PROD.

Also, `whitelist` / `blacklist` are terrible phrases!